packages/perseus/src/util.ts

```diff
@@ -270,10 +270,6 @@ function getGridStep(
     });
 }
 
-function snapStepFromGridStep(gridStep: [number, number]): [number, number] {
```

Comment:
Nice. I've always thought this `util.ts` file was too much of a grab-bag of stuff.  Nice to start pulling things apart into more focused files.